### ISSUE TITLE
Additional filters for super-admin user index

### ISSUE THREAD

**Issue by xsrust** [DMPRoadmap/roadmap#2352]
We recently had a bug report from an org-admin user who was missing a permission.  The account associated with the email address they had contacted us on had full org-admin perms.  After some digging on the Rails Console, it turned out the user had 2 different accounts, based off different email aliases, and they logged in with shibboleth credentials, so were unnaware of which account they were logging into.

I first attempted to interrogate this issue through our UI, but was quickly stopped by our super-admin `users` index page.  While I could sort by organisation, I could not filter to only see one organisation's users.  Further, I could not sort the users by the ones with permissions, to enable me to see how many admin users there are for a given organisation.

The organisational-filter, along with either a sort by permissions, or a filter for permissions (to easily see all org/super-admin users on your instance) would have been helpful.

**Comment by mariapraetzellis** [comment]
Agreed, works well and very useful. Thank you @xsrust and @martaribeiro!
